.rubocop_todo.yml

```diff
@@ -287,7 +287,7 @@ Metrics/BlockNesting:
 # Offense count: 3
 # Configuration parameters: CountComments.
 Metrics/ClassLength:
-  Max: 149
+  Max: 150
```

Comment:
Why is this in this PR?

Reply:
As the class required an extra line (the `RUBY_VERSION` line in the payload) and we didn't feel arbitrary limits disallowing new lines to be added (or requiring other lines be removed) without proper thought into refactoring is a good idea.